unit_test/batched/Test_Batched_VectorArithmatic.hpp

```diff
@@ -86,24 +86,24 @@ namespace Test {
         for (int k=0;k<vector_length;++k) 
           EXPECT_NEAR_KK( c[k], b[k]*alpha, eps);      
       }
-      {
-        /// test : vec / vec
-        c = a / b;
-        for (int k=0;k<vector_length;++k) 
-          EXPECT_NEAR_KK( c[k], a[k]/b[k], eps);      
+      // {
```

Comment:
This is fine, but I'm just checking to make sure you want this commented-out code.